blueprints/addon/files/.github/workflows/ci.yml

```
@@ -0,0 +1,65 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request: {}
+
+jobs:
+
+  lint:
+    name: "Lint"
+    env:
+      CI: true
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install node
+      uses: actions/setup-node@v2
+      with:
+        node-version: 12.x
+
+    - uses: actions/cache@v2
+      with:
+        path: <%= yarn ? '${{ steps.yarn-cache-dir-path.outputs.dir }}' : '~/.npm' %>
+        key: ${{ runner.os }}-modules-${{ hashFiles('<%= yarn ? '**/yarn.lock' : '**/package-lock.json' %>') }}
+    - name: Install Dependencies
+      run: <%= yarn ? 'yarn' : 'npm' %> install
+    - name: Lint
+      run: <%= yarn ? 'yarn' : 'npm run' %> lint
+
+  floating:
+    name: "Floating dependencies"
```

This isn't really about floating dependencies, it is about ember-try scenarios:

```suggestion
try-scenarios:
    name: ${{ matrix.try-scenario }}
```